packages/sns/src/errors/common.errors.ts

```diff
@@ -0,0 +1,7 @@
+// This is possible specially in SNS projects/
+// Because they share the same canisters but in different versions.
+export class UnsupportedMethodError extends Error {
+  constructor(public methodName: string) {
```

Comment:
Maybe make it `readonly` as well?

Reply:
good point

Reply:
But not good enough to actually do it?

Reply:
Oh, no! I thought I pushed the changes. My bad.